### PR TITLE
Uploaders/support multiple files

### PR DESCRIPTION
**Pull request by ugogo**

### CODE REVIEW

File: packages/react/src/components/FileUploader/index.js [hunk 1]
```diff
@@ -25,45 +25,54 @@ const texts = {
...
-    setInnerFile(eventFile);
-    safeInvoke(onChange(eventFile));
+    setFiles(nextFiles);
+    safeInvoke(onChange(inputFiles));
   };
 
-  const handleResetState = () => {
-    setInnerFile(undefined);
+  const handleDeleteFile = fileToDelete => {
+    const filteredFiles = files.filter(file => file.name !== fileToDelete.name);
```
Comment (pppemo) [inline packages/react/src/components/FileUploader/index.js]:
```suggestion
    const remainingFiles = files.filter(file => file.name !== fileToDelete.name);
```
?

No strong opinion though

File: packages/react/src/components/FileUploader/index.stories.js [hunk 2]
```diff
@@ -19,13 +18,18 @@ stories.add("Playground", () => (
     onChange={action("onChange")}
     isLoading={boolean("isLoading", false)}
     hasError={boolean("hasError", false)}
+    multiple={boolean("has multiple files", true)}
```
Comment (pppemo) [inline packages/react/src/components/FileUploader/index.stories.js]: 🌷`Has multiple files?`, `Has error`, `Is loading?`, etc.. At least I've been doing it this way in other components :)

Comment (ugogo) [reply packages/react/src/components/FileUploader/index.stories.js]: Yep, good point.
Maybe we should have guidelines for that too 🤔

File: packages/react/src/components/FileUploader/index.js (hunk 1 above)
Comment (ugogo) [reply packages/react/src/components/FileUploader/index.js]: ✅

**Pull request merged**